docs/reference-docs/api/workflow/rate-limit-workflow.md

```diff
@@ -1,21 +1,28 @@
 # Rate Limit Workflow
 
-The **workflow rate limit** limits the number of workflow executions at any given time. Workflow triggered after exceeding the rate limit will get queued based on the trigger time.
+The **workflow rate limit** limits the number of workflow executions at any given time. 
+Workflow triggered after exceeding the rate limit will get queued based on the trigger time.
+RateLimit config is a part of the workflow definition. The key can be parameterized.
 
-## API Endpoint
+## Java Example
+```shell
+        RateLimitConfig rateLimitConfig = new RateLimitConfig();
+        rateLimitConfig.setRateLimitKey("http");
```

Comment:
can we give a different name -- http is very common and I am 100% sure many will get confused here.